core/src/main/kotlin/com/strumenta/kolasu/symbolresolution/Symbols.kt

```diff
@@ -0,0 +1,6 @@
+package com.strumenta.kolasu.symbolresolution
+
+import com.strumenta.kolasu.model.PossiblyNamed
+
+interface Symbol : PossiblyNamed
```

Comment:
Note: I think that in Kolasu 2.0 a symbol could become a LIonWeb's Node (but not necessarily a Kolasu ASTNode, which will implement Node)

Reply:
Make sense! I think we should also think about extending the current symbol resolution mechanisms to support non-name-based symbol references, e.g. unnamed predicated references like 'the nearest enclosing class' or similar.